src/test/java/org.opentripplanner.middleware/OtpMiddlewareTest.java

```diff
@@ -0,0 +1,31 @@
+package org.opentripplanner.middleware;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.opentripplanner.middleware.spark.Main;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+/**
+ * This abstract class contains is used to start a test instance of datatools-server that other tests can use to perform
+ * various tests.
+ *
+ * A majority of the test involves setting up the proper config files for performing tests - especially the e2e tests.
+ */
+public abstract class OtpMiddlewareTest {
+    private static final Logger LOG = LoggerFactory.getLogger(OtpMiddlewareTest.class);
+    private static boolean setUpIsDone = false;
+
+    @BeforeAll
```

Comment:
Do we need this annotation in this file since it has no tests?

Reply:
Yes, because we want it to run before any test that is run in a class that implements this abstract class. However, I think we should remove it from `PersistenceTest` because all it's doing is calling this setup method. I'll remove the `PersistenceTest#setup` method.